ReleaseProcessAutomation/Jira/JiraFunctionality.cs

```diff
@@ -55,6 +61,39 @@ public void CreateAndReleaseJiraVersion (SemanticVersion currentVersion, Semanti
     ReleaseVersion(currentVersionID, nextVersionID, squashUnreleased);
   }
 
+  public bool MoveClosedIssuesToNewVersion (SemanticVersion currentVersion, SemanticVersion nextVersion)
+  {
+    var currentJiraVersion = _jiraVersionCreator.FindVersionWithVersionNumber(currentVersion.ToString());
+    var nextJiraVersion = _jiraVersionCreator.FindVersionWithVersionNumber(nextVersion.ToString());
+    if(currentJiraVersion == null || nextJiraVersion == null)
+    {
+      _console.WriteLine($"Could not find current jira version '{currentVersion}' or next jira version '{nextVersion}'");
+      return false;
+    }
+    
+    var closedIssues = _jiraIssueService.FindAllClosedIssues(currentJiraVersion.id);
+    _jiraIssueService.MoveIssuesToVersion(closedIssues, currentJiraVersion.id, nextJiraVersion.id);
+    return true;
+  }
+
+  public IEnumerable<JiraToBeMovedIssue> GetClosedVersionsSummaryWithExactFixVersion (SemanticVersion version)
+  {
+    var currentVersions = _jiraVersionCreator.FindAllVersionsWithVersionNumber(version.ToString()).ToList() 
+                          ?? throw new InvalidOperationException($"Could not find version '{version}'.");
+
+    var fullVersion = _jiraVersionCreator.FindVersionWithVersionNumber(version.ToString()) 
+                      ?? throw new InvalidOperationException($"Could not find version '{version}'.");
+
+    currentVersions.Remove(currentVersions.Find(projectVersion => projectVersion.id.Equals(fullVersion.id))!);
```

Comment:
We could simplify this statement using `RemoveAll(...)` as the id should be unique and only match one element.